lms/djangoapps/discussion_api/views.py

```diff
@@ -75,27 +76,41 @@ class CourseTopicsView(DeveloperErrorViewMixin, APIView):
     **Example Requests**:
 
         GET /api/discussion/v1/course_topics/course-v1:ExampleX+Subject101+2015
+            ?topic_id=a29d117faf43c7c59eda7717e722e9d8f4a47ea8
 
     **Response Values**:
+        if topic_id is not given:
+            * courseware_topics: The list of topic trees for courseware-linked
+              topics. Each item in the list includes:
 
-        * courseware_topics: The list of topic trees for courseware-linked
-          topics. Each item in the list includes:
+                * id: The id of the discussion topic (null for a topic that only
+                  has children but cannot contain threads itself).
 
-            * id: The id of the discussion topic (null for a topic that only
-              has children but cannot contain threads itself).
+                * name: The display name of the topic.
+
+                * children: A list of child subtrees of the same format.
+
+            * non_courseware_topics: The list of topic trees that are not linked to
+                  courseware. Items are of the same format as in courseware_topics.
+
+        if topic_id is given:
+            * id: The id of the discussion topic.
 
             * name: The display name of the topic.
 
             * children: A list of child subtrees of the same format.
-
-        * non_courseware_topics: The list of topic trees that are not linked to
-              courseware. Items are of the same format as in courseware_topics.
     """
```

Comment:
@jcdyer: I agree with your feedback.  To get more explicit about the new endpoint, do you think we should include `topic` to make it more clear?:
```
/api/discussion/v1/course_topics/course-v1:ExampleX+Subject101+2015/topic/a29d117faf43c7c59eda7717e722e9d8f4a47ea8
```
Or follow your format of just adding the topic id?:
```
/api/discussion/v1/course_topics/course-v1:ExampleX+Subject101+2015/a29d117faf43c7c59eda7717e722e9d8f4a47ea8
```
I like the former, including `topic`, but wondering what others think.